trust_stores_observatory/store_fetcher/java_fetcher.py

```diff
@@ -18,170 +20,161 @@
 import tarfile
 
 
+class JrePackage:
```

Comment:
I extracted this logic from the fetcher class so it's easier to read/understand.